pkg/agent/proxy/proxier.go

```diff
@@ -613,6 +630,31 @@ func (p *proxier) installServices() {
 	}
 }
 
+func (p *proxier) GetMCSExportedServiceEndpointIP(serviceCIDRIPv4 *net.IPNet, endpointIP string) *agenttypes.ServiceGroupInfo {
```

Comment:
Why capitalize the method? Is it used outside the package?

Reply:
Fixed, thanks.